src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjects/DefinitionProject.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.FindSymbols.DependentProjects
+{
+    internal readonly struct DefinitionProject : IEquatable<DefinitionProject>
+    {
+        private readonly ProjectId? _sourceProjectId;
+        private readonly string? _assemblyName;
```

Comment:
Nullable or no? The constructor is taking a non-null assemblyName.

Reply:
fixed up.  not-nullable.  i think this was made nullable because it's a struct.  But i'm fine with having the actual annotations represent the concept that this struct should never have a default value used.  In the future hte language may make it possible to strongly assert that that can't happen.